frontend/src/lib/components/apps/svelte-select/lib/tailwind.css

```diff
@@ -127,4 +127,14 @@
 
 .list-item {
     @apply list-none;
-}
\ No newline at end of file
+}
+
+.textarea-resize-none {
```

Comment:
https://tailwindcss.com/docs/resize
resize-none

Reply:
Thanks! Fixed for both